maintainers/team-list.nix

```diff
@@ -891,4 +891,13 @@ with lib.maintainers; {
     shortName = "Xfce";
     enableFeatureFreezePing = true;
   };
+
+  zig = {
+    members = [
+      figsoda
```

Comment:
Why not include all of us in the team already? It's not as if this was not happening already.

Reply:
because I want explicit permission from each person

Reply:
Do you want to be added to `CODEOWNERS` as well?

Reply:
OK to me. I  am interested in Zig since, what, 0.8?

Reply:
added you to everything

Reply:
> because I want explicit permission from each person, being the maintainer of the zig compiler is a bit different from being part of the zig team

That being said, I vote for not inserting @andrewrk. As the developer of upstream, he has too much work already.